test/tabbable.test.js

```diff
@@ -40,117 +58,125 @@ describe('tabbable', function() {
     cleanupFixtures();
   });
 
-  it('basic', function() {
-    var actual = fixture('basic').getTabbableIds();
-    var expected = [
-      'tabindex-hrefless-anchor',
-      'input',
-      'select',
-      'href-anchor',
-      'textarea',
-      'button',
-      'tabindex-div',
-      'hiddenParentVisible-button',
-    ];
-    assert.deepEqual(actual, expected);
-  });
+  [
+    { name: 'window', func: fixture },
+    { name: 'iframe\'s window', func: fixtureWithIframe },
+  ].forEach(function (assertionSet) {
 
-  it('nested', function() {
-    var actual = fixture('nested').getTabbableIds();
-    var expected = [
-      'tabindex-div-2',
-      'tabindex-div-0',
-      'input',
-    ];
-    assert.deepEqual(actual, expected);
-  });
+    describe(assertionSet.name, function() {
+    it('basic', function() {
+        var actual = fixture('basic').getTabbableIds();
```

Comment:
Shouldn't this be using the function `assertionSet.getFixture` (or `func`)?

Reply:
Yup, thanks for catching this too.  (This is what I get for trying to rush through this PR on Friday!)  I'll have an update up shortly!